src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -15,15 +15,24 @@ import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
 
 import type { StepValidation } from './utilities/useValidation';
 
-interface ValidatedTextInputPropTypes extends TextInputProps {
+type ValidatedTextInputPropTypes = TextInputProps & {
   dataTestId?: string | undefined;
   ouiaId?: string;
   ariaLabel: string | undefined;
   helperText: string | undefined;
   validator: (value: string | undefined) => boolean;
   value: string;
   placeholder?: string;
-}
+};
+
+type ValidationResult = {
```

Comment:
I don't think this needs to be an object since it's just three strings. You can just do something like
```javascript
type ValidationResult = 'default' | 'success' | 'error'
```